chapter2/tests/t20/terraform/prod/prod.tfvars

```diff
@@ -0,0 +1,63 @@
+/* 
+  ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ env=specific configuration variables                                                                             │
+  └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+ */
+
+environment = "prod"
+
+/* 
+  ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ configuration variables                                                                                    │
+  └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+ */
+
+vpc_tags = {
+  Name = "default"
+}
+
+availability_zones = ["us-east-2a", "us-east-2b", "us-east-2c"]
+
+/* 
+  ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ fqdn vars                                                                                                        │
+  └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+ */
+
+site_url            = "tf-maxim-omelchenko.saritasa-camps.link"
```

Comment:
Shouldn't you use a different domain name for different envs? The same is true for staging. Now applying terraform in different envs will change base url to this new env (i.e. dev/prod/staging websites would be not static, because you have 1 DNS name for all of them)

Reply:
> Shouldn't you use a different domain name for different envs? The same is true for staging. Now applying terraform in different envs will change base url to this new env (i.e. dev/prod/staging websites would be not static, because you have 1 DNS name for all of them)

@kseniyashaydurova 
No, we were supposed to make only one dev environment.

Reply:
So we can remove that `prod` and `staging` or update domain ) to be consistent if we have it

Reply:
> So we can remove that `prod` and `staging` or update domain ) to be consistent if we have it

@kseniyashaydurova 
Yes, updated
Done